### ISSUE TITLE
Can't install due to `tiktoken-node@workspace:*`

### ISSUE THREAD

**Issue by josiah-roberts** [anysphere/priompt#5]
`yarn add @anysphere/priompt && yarn add @anysphere/priompt-preview --dev` fails with
```
error Couldn't find package "@anysphere/tiktoken-node@workspace:*" required by "@anysphere/priompt" on the "npm" registry.
```

**Comment by kdallen1** [comment]
same here:
```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*
```

**Comment by arvid220u** [comment]
Thank you for reporting! This is fixed by #8
